Contents/mods/Hydrocraft/media/scripts/Chemistry.txt

```diff
@@ -41,6 +44,7 @@ item HCGunPowder
         Type            = Normal,
         DisplayName     = Salt Powder,
         icon            = HCSaltpowder,
+		DisplayCategory	= CraftMine,
```

Comment:
maybe craftchem? kinda hard since i think also popular use is for salt shakers as a seasoning